common.go

```diff
@@ -490,6 +490,14 @@ func GetLanguageExtensions(language string) []string {
 	return data.ExtensionsByLanguage[language]
 }
 
+// GetLanguageID returns the ID for the language. IDs are assigned by GitHub.
+//
+// NOTE: The zero value (0) is a valid language ID, so this API has no way to
```

Comment:
Maybe it would be better to return -1 for invalid languages?

Reply:
I would be open to this. I was concerned about Enry not being in control of these IDs, but talked to @lildude about it and he pointed out that the [IDs are now generated by hashing the name](https://github.com/github/linguist/blob/e6860ab64b463e022294b0f71d468d5243687027/script/update-ids#L8-L10), so a negative number is not possible.

However, it doesn't feel very Go-like. I would probably prefer to do it the way that [map access works](https://golang.org/doc/effective_go#maps) and return `(int, bool)`.

Reply:
> I would probably prefer to do it the way that map access works and return (int, bool).

Sounds very reasonable. Will you submit the patch as well?

Reply:
(int, bool) also is ok with me :)

Reply:
Thanks for the feedback 🙇 I'll submit an update to this PR that adjusts the return type.